solana/modules/ntt-messages/src/ntt.rs

```diff
@@ -50,22 +51,37 @@ impl Readable for NativeTokenTransfer {
         let to = Readable::read(reader)?;
         let to_chain = Readable::read(reader)?;
 
+        if A::SIZE != Some(0) {
+            // if the size is explicitly zero, this is an empty payload message
+            // and the size field should be skipped
+            // TODO: same as below for ntt additional_payload
```

Comment:
This seems like a TODO to be fixed in a future PR. Should there be a longer sentence here indicating what needs to be done?

Reply:
This refers to a more verbose todo in `transceiver.rs` (and was copied from `ntt.rs`). Updating.